packages/settings-library/src/settings_library/utils_service.py

```diff
@@ -0,0 +1,53 @@
+""" Helpers to build settings for services with http API
+
+
+"""
+
+
+from pydantic.networks import HttpUrl
+
+from .basic_types import PortInt
+
+DEFAULT_AIOHTTP_PORT: PortInt = 8000
+DEFAULT_FASTAPI_PORT: PortInt = 8080
+
+
+class MixinServiceSettings:
```

Comment:
from what I can see this is not yet, is that correct?

Reply:
It will be used in the next PR to build consistently urls in settings that keep PORT, HOST-type of info